cmd/get/upgrade-paths.go

```diff
@@ -0,0 +1,291 @@
+package get
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	distroconf "github.com/sighupio/fury-distribution/pkg/apis/config"
+	"github.com/sighupio/furyctl/configs"
+	"github.com/sighupio/furyctl/internal/analytics"
+	"github.com/sighupio/furyctl/internal/app"
+	"github.com/sighupio/furyctl/internal/config"
+	"github.com/sighupio/furyctl/internal/git"
+	"github.com/sighupio/furyctl/internal/semver"
+	cobrax "github.com/sighupio/furyctl/internal/x/cobra"
+	execx "github.com/sighupio/furyctl/internal/x/exec"
+	"github.com/sighupio/furyctl/pkg/dependencies"
+	dist "github.com/sighupio/furyctl/pkg/distribution"
+	netx "github.com/sighupio/furyctl/pkg/x/net"
+	yamlx "github.com/sighupio/furyctl/pkg/x/yaml"
+)
+
+func NewUpgradePathsCmd() *cobra.Command {
+	var cmdEvent analytics.Event
+
+	upgradePathsCmd := &cobra.Command{
+		Use:   "upgrade-paths",
+		Short: "Get available upgrade paths for the kind and version defined in the configuration file or a custom one.",
+		Long: `Get available upgrade paths for the kind and version defined in the configuration file or a custom one. If a from version or kind are specified the command will give the upgrade path for those instaed.
+ Examples:
+ - furyctl get upgrade-paths                               	will show the available upgrade paths for the kind a distribution version defined in the configuration file (furyctl.yaml by default)
```

Comment:
"...will show the available upgrade paths for the kind **_and_** distribution..."

Reply:
thanks!